core/build.gradle

```diff
@@ -5,7 +5,9 @@ dependencies {
     compile(project(":resources"))
     compile(project(":telemetry-client"))
     compileOnly("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.3")
-    compile("com.fasterxml.jackson.module:jackson-module-kotlin:$jacksonVersion")
+    compile("com.fasterxml.jackson.module:jackson-module-kotlin:$jacksonVersion") {
```

Comment:
we should probably actually do this as a global configuration, something like :

configurations {
  runtime.exclude group: "org.jetbrains.kotlin"
  runtime.exclude group: "org.jetbrains.kotlinx" //or whatever the kotlinx thing is
}

Reply:
Switched to a variant of this,

```
jar tf build/distributions/aws-jetbrains-toolkit-1.12-SNAPSHOT-192.zip | grep kotlin
aws-jetbrains-toolkit/lib/jackson-module-kotlin-2.9.8.jar
```